bucket/clash-verge-rev.json

```diff
@@ -17,15 +17,15 @@
     "installer": {
         "script": [
             "ensure \"$persist_dir\" | Out-Null",
-            "New-Item \"$env:USERPROFILE\\.config\\clash-verge\" -ItemType Junction -Target \"$persist_dir\" | Out-Null"
+            "New-Item \"$env:APPDATA\\io.github.clash-verge-rev.clash-verge-rev\" -ItemType Junction -Target \"$persist_dir\" | Out-Null"
```

Comment:
Does the config of the old version not affect the new version of the software?

Also, @xyalter mentioned that the original approach of using "$persist_dir\.config" is still valid in https://github.com/ScoopInstaller/Extras/pull/14494#issuecomment-2522511538. I am not sure whether it can cause some related changes. Please check first.

Thank you.

Reply:
The config of the old version is stored in `$persist_dir\.config\io.github.clash-verge-rev.clash-verge-rev`, but the new version is directly stored in `$persist_dir` in my PR, so they should not be conflicted. Also, Clash-verge-rev recommends configuring it from scratch, using 2.x app with v1 config will cause many problems.

`$persist_dir\.config` is indeed valid. We can use the same portable persist logic, but should move/remove the old config.